lib/pipedrive/base.rb

```diff
@@ -73,15 +73,15 @@ def authenticate(token)
       #
       # @param [HTTParty::Response] response
       def bad_response(response, params={})
-        raise response.inspect
+        puts params.inspect
```

Comment:
puts

Reply:
yeah I know, it is the fucking gem written like so. I didn't want to do too many changes, but fuck it, I'll delete this mfker

Reply:
![image](https://media3.giphy.com/media/EtB1yylKGGAUg/200.gif)